client/src/components/MultiJurisdictionAudit/StatusBox/index.test.tsx

```diff
@@ -286,6 +286,25 @@ describe('StatusBox', () => {
     })
   })
 
+  it('shows audit name', () => {
```

Comment:
instead of adding a new test case for the audit name, i think it would be simpler to just add an assertion checking for the audit name to one of the existing test cases